editors/emacs/files/patch-inline-25.2-20170426.diff

```diff
@@ -802,40 +745,59 @@ diff -Nrc lisp/term/ns-win.el lisp/term/ns-win.el
 +   (define-key function-key-map [134219941] [?\M-\\])
 +   (define-key function-key-map [75497596] [?\C-\M-\\])
 +   (define-key function-key-map [201328805] [?\C-\M-\\])
-+ )
++   )
 + 
 + ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 + 
   (provide 'ns-win)
   
   ;;; ns-win.el ends here
-diff -Nrc src/Makefile.in src/Makefile.in
-*** src/Makefile.in	2016-06-29 18:49:20.000000000 +0900
---- src/Makefile.in	2016-09-20 02:27:22.000000000 +0900
+diff -crN emacs-25.2_orig/src/Makefile.in emacs-25.2/src/Makefile.in
+*** emacs-25.2_orig/src/Makefile.in	2017-02-03 19:25:44.000000000 +0900
+--- emacs-25.2/src/Makefile.in	2017-04-26 14:45:40.000000000 +0900
 ***************
-*** 409,416 ****
+*** 409,415 ****
   SOME_MACHINE_OBJECTS = dosfns.o msdos.o \
     xterm.o xfns.o xmenu.o xselect.o xrdb.o xsmfns.o fringe.o image.o \
     fontset.o dbusbind.o cygw32.o \
 !   nsterm.o nsfns.o nsmenu.o nsselect.o nsimage.o nsfont.o macfont.o \
-!   w32.o w32console.o w32fns.o w32heap.o w32inevt.o w32notify.o \
+    w32.o w32console.o w32fns.o w32heap.o w32inevt.o w32notify.o \
     w32menu.o w32proc.o w32reg.o w32select.o w32term.o w32xfns.o \
     w16select.o widget.o xfont.o ftfont.o xftfont.o ftxfont.o gtkutil.o \
-    xsettings.o xgselect.o termcap.o
---- 409,416 ----
+--- 409,415 ----
   SOME_MACHINE_OBJECTS = dosfns.o msdos.o \
     xterm.o xfns.o xmenu.o xselect.o xrdb.o xsmfns.o fringe.o image.o \
     fontset.o dbusbind.o cygw32.o \
-!   nsterm.o nsfns.o nsmenu.o nsselect.o nsimage.o nsfont.o macfont.o macim.o \
-!   w32.o w32console.o w32fns.o w32heap.o w32inevt.o \
+!   nsterm.o nsfns.o nsmenu.o nsselect.o nsimage.o nsfont.o macfont.o macim.o\
+    w32.o w32console.o w32fns.o w32heap.o w32inevt.o w32notify.o \
     w32menu.o w32proc.o w32reg.o w32select.o w32term.o w32xfns.o \
     w16select.o widget.o xfont.o ftfont.o xftfont.o ftxfont.o gtkutil.o \
-    xsettings.o xgselect.o termcap.o
-diff -Nrc src/keyboard.c src/keyboard.c
-*** src/keyboard.c	2016-08-02 17:06:01.000000000 +0900
---- src/keyboard.c	2016-09-20 02:27:22.000000000 +0900
+diff -crN emacs-25.2_orig/src/keyboard.c emacs-25.2/src/keyboard.c
+*** emacs-25.2_orig/src/keyboard.c	2017-04-20 23:50:42.000000000 +0900
+--- emacs-25.2/src/keyboard.c	2017-04-26 14:45:40.000000000 +0900
 ***************
-*** 3875,3882 ****
+*** 2472,2481 ****
+  	swallow_events (false);		/* May clear input_pending.  */
+  
+        /* Redisplay if no pending input.  */
+!       while (!(input_pending
+! 	       && (input_was_pending || !redisplay_dont_pause)))
+  	{
+- 	  input_was_pending = input_pending;
```

Comment:
You forgot to bump the revision when you changed files installed by the port, which (accidentally) caused the CI not to test it.

Reply:
Ah, good to know. Probably should document it somewhere.